components/EntryList/EntryList.js

```diff
@@ -1,58 +1,53 @@
-import { nanoid } from "nanoid";
-import { StyledEntryList } from "./EntryList.styled";
+import Animation from "../Animation/Animation";
 import Intensity from "@/utils/intensity";
+import { useState, useEffect } from "react";
+import * as Styled from "./EntryList.styled";
 
-import {
-  StyledEntry,
-  SliderText,
-  ExperienceText,
-  EntryText,
-  Separator,
-  EditDeleteButton,
-  StyledEntryHeadnote,
-} from "@/components/EntryList/EntryList.styled";
+export default function EntriesList() {
+  const [dataFromLocalStorage, setDataFromLocalStorage] = useState([]);
 
-export default function EntryList({ allEntries, deleteEntry }) {
-  function handleDeleteEntryListItem(id) {
-    deleteEntry(id);
+  useEffect(() => {
+    const dataFromLocalStorage = localStorage.getItem("anonymous_moods");
+    if (dataFromLocalStorage) {
+      const parsedData = JSON.parse(dataFromLocalStorage);
+      setDataFromLocalStorage(parsedData);
+    }
+  }, []);
+
+  function handleDeleteEntry(index) {
+    const updatedData = [...dataFromLocalStorage];
+    updatedData.splice(index, 1);
+    setDataFromLocalStorage(updatedData);
+    localStorage.setItem("anonymous_moods", JSON.stringify(updatedData));
```

Comment:
```suggestion
  function handleDeleteEntry(id) {
    const updatedMoods =  moods.filter()... <- write filter function here;
    setMoods(updatedMoods);
```

I would recommend you create an ID for every entry the user creates. The Id would also be used to filter instead of splice. 
When you filter via the id it is much more secure that you delete the right entry. (thinking longterm, as you might work more with the array of moods)